scripts/specializations/CpAIFieldWorker.lua

```diff
@@ -354,9 +354,20 @@ function CpAIFieldWorker:replaceAIFieldWorkerDriveStrategies(jobParameters, star
         local combine = AIUtil.getImplementOrVehicleWithSpecialization(self, Combine) 
         local pipe = combine and SpecializationUtil.hasSpecialization(Pipe, combine.specializations)
         if combine and pipe then -- Default harvesters with a pipe.
-            CpUtil.debugVehicle(CpDebug.DBG_FIELDWORK, self, 'Found a combine with pipe, install CP combine drive strategy for it')
-            cpDriveStrategy = AIDriveStrategyCombineCourse.new()
-            self.spec_cpAIFieldWorker.combineDriveStrategy = cpDriveStrategy
+            local capacity = 0
```

Comment:
This check should be implemented in the chopper strategy as an isChopper() function and not here.